src/commands/dev.ts

```diff
@@ -250,7 +263,21 @@ export async function command(commandOptions: CommandOptions) {
         }
         return `/web_modules/${spec}.js`;
       });
+
+      if (!fileLoc.includes('web_modules') && !fileLoc.includes('node_modules')) {
+        const imports = await scanCodeImportsExports(builtFileResult.result);
+        for (const imp of imports.reverse()) {
+          let spec = builtFileResult.result.substring(imp.s, imp.e);
+          if (imp.d > -1) {
```

Comment:
oh this is interesting: do we bubble up dynamic imports as well? I'm not sure if I understand how that would work, which means I'd prefer we don't support it in v1 until we have a user bring us a use case that we can understand.